diff -up CGNS-4.5.0/src/CMakeLists.txt.1~ CGNS-4.5.0/src/CMakeLists.txt
--- CGNS-4.5.0/src/CMakeLists.txt.1~	2025-02-12 03:43:11.689512857 +0100
+++ CGNS-4.5.0/src/CMakeLists.txt	2025-02-12 03:44:12.895413336 +0100
@@ -703,8 +703,8 @@ endif ()
 # for windows, need to install both cgnsdll.dll and cgnsdll.lib
 install (TARGETS ${install_targets}
          EXPORT cgns-targets
-         LIBRARY DESTINATION lib COMPONENT libraries
-         ARCHIVE DESTINATION lib COMPONENT libraries
+         LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT libraries
+         ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT libraries
          RUNTIME DESTINATION bin COMPONENT libraries
          INCLUDES DESTINATION include)
 
@@ -756,14 +756,14 @@ write_basic_package_version_file(
 configure_package_config_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/cgns-config.cmake.in
   ${CMAKE_CURRENT_BINARY_DIR}/cgns-config.cmake
-  INSTALL_DESTINATION lib/cmake/cgns )
+  INSTALL_DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/cgns" )
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cgns-config-version.cmake
               ${CMAKE_CURRENT_BINARY_DIR}/cgns-config.cmake
-	DESTINATION lib/cmake/cgns)
+	DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/cgns")
 install(EXPORT cgns-targets
 	FILE cgns-targets.cmake
 	NAMESPACE CGNS::
-	DESTINATION lib/cmake/cgns
+	DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/cgns"
 	)
 #endif()
 
