--- CGNS-3.3.1/src/CMakeLists.txt.orig	2017-10-10 22:52:54.240429994 +0200
+++ CGNS-3.3.1/src/CMakeLists.txt	2017-10-10 22:53:43.505861113 +0200
@@ -501,10 +501,11 @@
 option(CGNS_USE_SHARED "Link programs to the CGNS shared library" "ON")
 
 # Build a static version of the library
+
 if (CGNS_ENABLE_FORTRAN)
-    add_library(cgns_static ${cgns_FILES} $<TARGET_OBJECTS:cgns_f2c>)
+  add_library(cgns_static STATIC ${cgns_FILES} $<TARGET_OBJECTS:cgns_f2c>)
 else (CGNS_ENABLE_FORTRAN)
-  add_library(cgns_static ${cgns_FILES})
+  add_library(cgns_static STATIC ${cgns_FILES})
 endif (CGNS_ENABLE_FORTRAN)
 
 
